packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -465,11 +469,23 @@ class FlexibleSpaceBarSettings extends InheritedWidget {
   /// these elements upon initialization.
   final double currentExtent;
 
+  /// True if the FlexibleSpaceBar overlaps the primary scrollable's contents.
+  ///
+  /// This value is used by the [AppBar] to resolve
+  /// [AppBar.backgroundColor] against [MaterialState.scrolledUnder],
+  /// i.e.  to enable apps to specify different colors when content
+  /// has been scrolled up and behind the app bar.
+  ///
+  /// Null if the called hasn't determined if the FlexibleSpaceBar
```

Comment:
I think this might be a typo?

```suggestion
  /// Null if the caller hasn't determined if the FlexibleSpaceBar
```

Reply:
Good catch, fixed.

Reply:
Ok